mixer/pkg/runtime2/dispatcher/session.go

```diff
@@ -30,7 +30,7 @@ const queueAllocSize = 64
 
 // session represents a call session to the Dispatcher. It contains all the mutable state needed for handling the
 // call. It is used as temporary memory location to keep ephemeral state, thus avoiding garbage creation.
-type session struct {
+type session struct { // nolint: maligned
```

Comment:
This is a runtime data structure, we should fix this warning by rearranging the fields.

Reply:
Okay... fixed.